src/main/java/uk/gov/companieshouse/api/testdata/model/entity/CompanyProfile.java

```diff
@@ -5,26 +5,165 @@
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 
+import java.time.Instant;
+import java.util.List;
+
 @Document(collection = "company_profile")
 public class CompanyProfile {
 
+    public class Accounts {
+        @Field("next_due")
+        private Instant accountsNextDue;
+        @Field("next_accounts.period_start_on")
+        private Instant periodStart;
+        @Field("next_accounts.period_end_on")
+        private Instant periodEnd;
+        @Field("next_accounts.due_on")
+        private Instant nextAccountsDueOn;
+        @Field("next_accounts.overdue")
+        private Boolean nextAccountsOverdue;
+        @Field("next_made_up_to")
+        private Instant accountsNextMadeUpTo;
```

Comment:
Can we review the names of these fields to shorten them by removing `accounts` when redundant?